### PR TITLE
New version: Agilent.LabAdvisor version 2.21.597

### PR DESCRIPTION
**Pull request by Spacefish**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199122)

### CODE REVIEW

**Comment by Spacefish** [comment]
Product page: https://www.agilent.com/en/product/liquid-chromatography/hplc-ce-software/instrument-control-software/lab-advisor-software is correct 403 is probably caused by some rate limits / security software blocking the request from the build slave.. Not sure.

**Comment by stephengillie** [comment]
It might be a data center block on network connections, to prevent automated downloads.

**Comment by Spacefish** [comment]
> It might be a data center block on network connections, to prevent automated downloads.
Maybe it was just rate-limited or something like that.. Can we just run the pipeline one more time and see if it works?

**Comment by Trenly** [comment]
> > It might be a data center block on network connections, to prevent automated downloads.
> 
> Maybe it was just rate-limited or something like that.. Can we just run the pipeline one more time and see if it works? It´s not he download URL but rather the product info page on the website.

Yeah, we can run it one more time. Alternatively, you could just comment out that URL from the locale file

File: manifests/a/Agilent/LabAdvisor/2.21.597/Agilent.LabAdvisor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: Agilent.LabAdvisor
+PackageVersion: 2.21.597
+PackageLocale: en-US
+Publisher: Agilent Technologies
+Author: Agilent
+PackageName: Agilent Lab Advisor
+PackageUrl: https://www.agilent.com/en/product/liquid-chromatography/hplc-ce-software/instrument-control-software/lab-advisor-software
```
Comment (stephengillie) [inline manifests/a/Agilent/LabAdvisor/2.21.597/Agilent.LabAdvisor.locale.en-US.yaml:7]:
```suggestion
# PackageUrl: https://www.agilent.com/en/product/liquid-chromatography/hplc-ce-software/instrument-control-software/lab-advisor-software
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**